packages/web-react/src/components/Breadcrumbs/useBreadcrumbsStyleProps.tsx

```diff
@@ -0,0 +1,21 @@
+import { ElementType } from 'react';
+import { useClassNamePrefix } from '../../hooks/useClassNamePrefix';
+import { SpiritBreadcrumbsProps, BreadcrumbsProps } from '../../types';
+
+export interface BreadcrumbsStyles {
+  /** className props */
+  classProps: string;
+  /** props to be passed to the element */
+  props: BreadcrumbsProps;
+}
+
+export function useBreadcrumbsStyleProps<T extends ElementType = 'nav'>(
+  props: SpiritBreadcrumbsProps<T>,
```

Comment:
```suggestion
  props: BreadcrumbsProps,
```

Reply:
Yes, that's what I meant in comment above.